### PR TITLE
added changes to ride form

### PR DESCRIPTION
**Pull request by LouieAldana**
- unchangeable user name 
- rider_radius slider with min-max
- formatted contact info (number)
- mui calendar

### CODE REVIEW

File: starter-code/client/src/pages/Ride.js [hunk 1]
```diff
@@ -450,28 +501,98 @@ export default function Ride() {
...
-            autoComplete="off"
-            disabled={!shouldPickup}
-          />
-          {errorMsgs.rider_radius && (
-            <p className="error-msg">{errorMsgs.rider_radius}</p>
+          {shouldPickupRiders && (
+            <div>
+              How far are you willing to pickup riders from your start location
+              (in meters)
+              <Tooltip title="1600 meters = 0.99 miles">
```
Comment (LucienLuc) [inline starter-code/client/src/pages/Ride.js:532]: Add question mark after statement. And you can probably just round the 0.99 miles to 1.

File: starter-code/client/src/pages/Ride.js [hunk 2]
```diff
@@ -450,28 +501,98 @@ export default function Ride() {
...
+                  valueLabelDisplay="auto"
+                />
+              </div>
+              {errorMsgs.rider_radius && (
+                <p className="error-msg">{errorMsgs.rider_radius}</p>
+              )}
+            </div>
           )}
+          <div>
+            Do you want to enter your contact Information?
```
Comment (LucienLuc) [inline starter-code/client/src/pages/Ride.js:553]: Lowercase on "information".

**Review by LucienLuc — CHANGES_REQUESTED** [review]

File: starter-code/client/src/pages/Ride.js (hunk 1 above)
Comment (LouieAldana) [reply starter-code/client/src/pages/Ride.js]: done

File: starter-code/client/src/pages/Ride.js (hunk 2 above)
Comment (LouieAldana) [reply starter-code/client/src/pages/Ride.js]: done

**Pull request merged**